tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -155,17 +163,28 @@ private static void handleDescribeReplication(Admin admin) throws ExecutionExcep
         );
     }
 
-    private static List<List<String>> quorumInfoToRows(QuorumInfo.ReplicaState leader, Stream<QuorumInfo.ReplicaState> infos, String status) {
-        return infos.map(info ->
-            Stream.of(
+    private static List<List<String>> quorumInfoToRows(QuorumInfo.ReplicaState leader,
+                                                       Stream<QuorumInfo.ReplicaState> infos,
+                                                       String status,
+                                                       boolean readable) {
```

Comment:
```suggestion
                                                       boolean humanReadable) {
```
nit: suggestion to change `readable` to `humanReadable`, if accepted, it has to be applied throughout: the caller and the callee of this method.